pypdf/_writer.py

```diff
@@ -497,13 +497,13 @@ def _add_page(
         else:
             cast(ArrayObject, node[PA.KIDS]).append(page.indirect_reference)
             self.flattened_pages.append(page)
-        cpt = 1000
+        cnt = 1000
```

Comment:
Are we able to find an even better name here? For me, neither `cpt` nor `cnt` are really self-explaining.

Reply:
Have changed to "recurse", there may be better.